mmdet/models/roi_heads/grid_roi_head.py

```diff
@@ -154,7 +154,7 @@ def simple_test(self,
             num_imgs = len(det_bboxes)
             for i in range(num_imgs):
                 if det_bboxes[i].shape[0] == 0:
-                    bbox_results.append(grid_rois.new_tensor([]))
+                    bbox_results.append([np.zeros((0, 5), dtype=np.float32) for i in range(self.bbox_head.num_classes)])
```

Comment:
Why it is numpy array rather than tensor? can we use `grid_rois_new_tensor(size=(0,5))`?